### PR TITLE
Changes needed for latest NTA load (UA-799)

### PR DESCRIPTION
**Pull request by kotodharma**

### CODE REVIEW

File: app/models/nta_upload.rb [hunk 1]
```diff
@@ -295,10 +295,10 @@ def make_country_geos(row_data)
                                display_name_short: row_data['subregn'],
                                geotype: 'region2',
                                parents: geo_region.id })
-    income_grp = (row_data['incgrp'] || row_data['incgrp2015']).sub(/.income$/i, '').titlecase
+    income_grp = (row_data['incgrp'] || row_data['incgrp2015']).sub(/\s*countries.*$/i, '').titlecase
     geo_incgrp = Geography.get_or_new_nta({ handle: income_grp.gsub(/\W+/, '_') },
-                                          { display_name: "#{income_grp} Income",
-                                            display_name_short: "#{income_grp} Income",
+                                          { display_name: "#{income_grp} Countries",
```
Comment (jonpage) [inline app/models/nta_upload.rb]: Does this add "Countries" anywhere they didn't.

File: app/models/nta_upload.rb [hunk 2]
```diff
@@ -496,15 +506,15 @@ def load_data_postproc
...
-              when 'Africa' then 1
-              when 'Americas' then 2
-              when 'Asia' then 3
-              when 'Europe' then 4
-              when 'Oceania' then 5
-              when 'Low Income' then 7
-              when 'Lower Middle Income' then 8
-              when 'Upper Middle Income' then 9
-              when 'High Income' then 10
+              when 'African Region' then 1
```
Comment (jonpage) [inline app/models/nta_upload.rb]: The wanted this to be "Countries" not "Region"

File: app/models/nta_upload.rb (hunk 1 above)
Comment (kotodharma) [reply app/models/nta_upload.rb]: No, that's their change.

File: app/models/nta_upload.rb (hunk 2 above)
Comment (kotodharma) [reply app/models/nta_upload.rb]: Well, yeah, but... this is now a Geography, and the "region" series (Asia, Americas, etc) are connected to it, so... it seemed more "logical" :=) to me to word it this way. If you think I should just change it back, it's easy enough.

Comment (kotodharma) [reply app/models/nta_upload.rb]: 'Course... now that I consider it.. the income pseudo-region groupings are called "blah countries", so I guess it's more consistent that way. Will change.

Comment (jonpage) [reply app/models/nta_upload.rb]: If the naming here does not affect the display on the data portal, feel free to go with your gut.

**Pull request merged**